src/lib/seam/connect/api-error.ts

```diff
@@ -0,0 +1,30 @@
+// UPSTREAM: Should be provided by @seamapi/types.
+interface ApiError {
+  type: string
+  message: string
+  data?: unknown
+}
+
+export class SeamHttpApiError extends Error {
+  code: string
+  statusCode: number
```

Comment:
Could be `status`, but IMO `statusCode` is unambiguous and the standard name: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status